software/caddy/en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "caddy"
+lastmod = "2021-02-17T16:39:55-03:00"
++++
+# caddy
+
+Solus provides the caddy web server and proxy to users and developers who want to test their web applications before setting them to production.
+
+Remember that Solus is not a server operating system: any usage outside local testing is discouraged.
+
+## Installation
+
+caddy is available in the `caddy` package. It can be installed from the [Software Center](/articles/common-tasks/installing-and-updating-software/en/) or via terminal:
+
+    ```bash
+    sudo eopkg install caddy
+    ```
```

Comment:
Shouldn't be indenting and you never fixed the missing whitespace after `bash`

Reply:
Yeah, seems I've forgot about the Markdown syntax for selecting the language